warehouse/malware/checks/base.py

```diff
@@ -43,12 +57,12 @@ def backfill(self, sample=1):
         """
 
     def _load_check_id(self):
-        self.id = (
+        (self.id,) = (
```

Comment:
Is this necessary? I think the parenthesis here are for a multiline continuation of the `self.db.query(MalwareCheck.id)` statement, and not a tuple.

Reply:
I didn't think it was, but `black` complained about it. (This was the CI lint failure that was confusing me, since my local `black` hadn't complained about it until I ran a full `make lint`).